src/persistentStore.ts

```diff
@@ -6,11 +6,35 @@
 
 import os from 'os';
 import path from 'path';
-import { getPersistentStore as store } from 'pc-nrfconnect-shared';
+import {
+    getPersistentStore as store,
+    logger,
+    usageData,
+} from 'pc-nrfconnect-shared';
 
 import getNrfutilConfig from './Manager/nrfutil/config';
+import nrfutilToolchainManager from './Manager/nrfutil/nrfutilToolchainManager';
 
-export const defaultInstallDir = getNrfutilConfig().install_dir;
+// Set fallback value in case nrfutil can't supply the value.
+let installDir =
+    {
+        win32: path.resolve('\\', 'ncs'),
```

Comment:
This line currently means `\ncs` in the current drive. So e.g. if people installed the app on drive E:, this would be `E:\ncs`. I don't think, this is really intended.

[In `nrfutil-toolchain-manager` the default dir is `C:\ncs`](https://github.com/NordicPlayground/nrfutil-rs/blob/897353594c6c5b2539904c1d0b5d65a016ca11f6/nrfutil-toolchain-manager/src/config.rs#L157).

This is probably better:
```suggestion
        win32: path.resolve('C:', 'ncs'),
```